lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/cw/XwikiController.java

```diff
@@ -147,6 +148,14 @@ public InputStream getModel(String modelSetId, ModelSetType type) throws LpRestE
     public String startSimulation(String modelId, String currentUser, Collection<UserData> potentialUsers)
         throws LpRestException
     {
+    	currentUser = this.removePrefixes(currentUser);
```

Comment:
Yes, this could be eventually removed. In principle, the CP does not have any need to worry about that. That was just an "additional" operation I added inside that Controller, as I don't think that it would create problems. Can we keep it?